src/src/beneficiaries/beneficiaries.service.ts

```diff
@@ -171,13 +173,59 @@ export class BeneficiariesService {
 		try {
 			const user = await this.userService.ipUserInfo(req);
 
+			const variables: any = {};
+
+			let filterQueryArray = [];
+			let paramsQueryArray = [];
+
+			if (body.search && body.search !== '') {
+				 let first_name = body.search.split(" ")[0]
+				 let last_name = body.search.split(" ") ? body.search.split(" ")[1] : ""
```

Comment:
Replace this line with `let last_name = body.search.split(" ")[1] || "";`

Reply:
resolved comment